src/server/routes/api/document/reach.js

```diff
@@ -299,8 +299,13 @@ module.exports = {
                       return INTERACTION_TYPE.PHOSPHORYLATION;
                     case REACH_EVENT_TYPE.METHYLATION:
                       return INTERACTION_TYPE.METHYLATION;
-                    default:
-                      return null;
+                  }
+                }
+                else if ( argsAreEnts ) {
+                  // more cases would be added in the future
+                  switch ( frame.type ) {
+                    case REACH_EVENT_TYPE.ACTIVATION:
+                      return INTERACTION_TYPE.MODIFICATION;
```

Comment:
Only the subclasses of modification are used in the UI.  If it's not something like phosphorylation, then it should probably be general.